InvenTree/stock/models.py

```diff
@@ -1651,6 +1652,17 @@ def before_delete_stock_item(sender, instance, using, **kwargs):
         child.save()
 
 
+@receiver(post_save, sender=StockItem, dispatch_uid='stock_item_post_save_log')
+def after_save_stock_item(sender, instance: StockItem, **kwargs):
+    """
+    Check if the stock quantity has fallen below the minimum threshold of part. If yes, notify the users who have
+    starred the part
+    """
+
+    if instance.quantity <= instance.part.minimum_stock:
+        inventree_tasks.notify_low_stock(instance)
```

Comment:
Maybe this should be called as Async_task? Thinking of allocating etc . WDYT @SchrodingersGat ?

Reply:
@matmair I thought of making it an AsyncTask but made it a sequential one since the `offload_task()` function which eventually gets called by `notify_low_stock()` runs the task asynchronously .

Reply:
It is still a db-access but we can come back to this if we see perf problems with stock changes,

Reply:
I think offload_task is OK to use here